owncloudApp/src/main/java/com/owncloud/android/MainApp.java

```diff
@@ -264,18 +233,25 @@ public static String getUserAgent() {
         return String.format(appString, version);
     }
 
-    public static boolean isBeta() {
-        boolean isBeta = false;
-        try {
-            String packageName = getAppContext().getPackageName();
-            PackageInfo packageInfo = getAppContext().getPackageManager().getPackageInfo(packageName, 0);
-            String versionName = packageInfo.versionName;
-            if (versionName.contains(BETA_VERSION)) {
-                isBeta = true;
-            }
-        } catch (PackageManager.NameNotFoundException e) {
-            e.printStackTrace();
+    public static boolean isDeveloper() {
+        return mDeveloper;
+    }
+
+    public void readIsDeveloper() {
+        mDeveloper = BuildConfig.DEBUG || PreferenceManager.getDefaultSharedPreferences(getApplicationContext())
+                .getInt(CLICK_DEV_MENU, 0) > CLICKS_NEEDED_TO_BE_DEVELOPER;
```

Comment:
Use a constant for the default clicks value = 0

Reply:
Now we have `private static final int DEFAULT_0 = 0;`

but the line below makes me much more doubt

![image](https://user-images.githubusercontent.com/3314607/54336572-b6269d00-462c-11e9-967c-30170d669eea.png)

Reply:
Maybe you do not need to call it `DEFAULT_0` but `CLICKS_DEFAULT` since you are comparing it with `CLICKS_NEEDED_TO_BE_DEVELOPER`